hera_cal/tests/test_lstbin.py

```diff
@@ -29,15 +28,16 @@ def setup_method(self):
                            sorted(glob.glob(DATA_PATH + '/zen.2458044.4*XRAA.uvh5')),
                            sorted(glob.glob(DATA_PATH + '/zen.2458045.4*XRAA.uvh5'))]
 
-        (self.data1, self.flgs1, self.ap1, a, self.freqs1, t, self.lsts1,
-         p) = io.load_vis(self.data_files[0], return_meta=True, filetype="uvh5")
-        (self.data2, self.flgs2, ap, a, self.freqs2, t, self.lsts2,
-         p) = io.load_vis(self.data_files[1], return_meta=True, filetype="uvh5")
-        (self.data3, self.flgs3, ap, a, self.freqs3, t, self.lsts3,
-         p) = io.load_vis(self.data_files[2], return_meta=True, filetype="uvh5")
+        (self.data1, self.flgs1, self.nsmps1, self.ap1, a, self.freqs1, t, self.lsts1,
+         p) = io.load_vis(self.data_files[0], return_meta=True, filetype="uvh5", return_nsamples=True)
```

Comment:
Is the only place you want to use the new features in load_vis? If so, can we just eliminate the use of load_vis?

Reply:
I've replaced this with `HERAData` convention.